.github/ISSUE_TEMPLATE/issue_template.yml

```diff
@@ -0,0 +1,71 @@
+name: Cadastro de Palestra
+description: Use este template para cadastrar sua palestra.
+title: "[PALESTRA] "
+labels: ["palestra"]
+assignees: []
+
+body:
+  - type: input
+    id: titulo
+    attributes:
+      label: Título
+      description: Título descritivo e interessante.
+      placeholder: "Título da palestra"
+      #required: true
```

Comment:
Se o required default é true, acho que quando o código está comentado talvez faça mais sentido remover. Digo deste e dos outros required.

Reply:
Eu preferi comentar pois segundo a documentação: 
"The required field key is only supported in public repositories. In private and internal repositories, all fields are optional."
No fork ele gerou um erro e quebrou o template, mas acredito que pode ser ajustado no repositório principal por conta desta particularidade após o merge.

Reply:
Falha minha!
Revisei e atualizei as estruturas de validação de acordo com a documentação.
Estava faltando uma parte da estrutura:
validations:
      required: true